### ISSUE TITLE
refactoring_sandbox : VP2 Delegate : Moving Proxy Shape Does Not Update Prim Positions in Viewport

### ISSUE THREAD

**Issue by cfmoore007** [Autodesk/maya-usd#71]
**Issue :**

Moving the proxyShape, does not update prim positions in the viewport

**Steps to re-produce :**
```
import maya.cmds as cmds
cmds.loadPlugin ("AL_USDMayaPlugin")

# import proxyShape
cmds.AL_usdmaya_ProxyShapeImport(file="3_Spheres.usda")

# select proxy shape
cmds.select("AL_usdmaya_Proxy")

# notice, nothing in the viewport is moved
cmds.move(-10,rpr=True,z=True)
```

**Testing Environment :**
(Linux) Maya 2019.2
refactoring_sandbox ( d0bb624 ) and Pixar USD dev ( b85ddac )

( Windows ) beta ( latest PR )
refactoring_sandbox ( w/ PR#69 ) and Pixar USD-rc3

( having issues uploading example file, sorry! )

**Comment by BigRoy** [comment]
Are you able to reproduce this issue with the Hydra viewport too? I mentioned this bug too in #51 and it seems to be the same behavior. Note that I did not use the AL plug-in but the proxy shape from the maya usd plug-in itself.

**Comment by cfmoore007** [comment]
In the older refactoring_sandbox (737616d), yes.  The newer I'm having some issues with mtoh ... and getting some warnings.  ( Honestly, we're stress testing the VP2 delegate more ).

**Comment by BigRoy** [comment]
> The newer I'm having some issues with mtoh ... and getting some warnings. 

These are likely fixed with https://github.com/Autodesk/maya-usd/pull/79 just now. However that does not fix your VP2 issue reported here.

**Comment by cfmoore007** [comment]
[3_Spheres.zip](https://github.com/Autodesk/maya-usd/files/3773891/3_Spheres.zip)
added file

**Comment by HdC-adsk** [comment]
Hi @cfmoore007 Thanks for raising this issue. I am looking into it.

**Comment by HdC-adsk** [comment]
Logged as MAYA-99904 internally.

**Comment by cfmoore007** [comment]
This seems addressed in PR#112

**Comment by HdC-adsk** [comment]
Yes, the issue has been fixed by #112 . Thanks @cfmoore007 .
